bugbug/bug_features.py

```diff
@@ -896,3 +897,235 @@ def __call__(
             for is_type in self.bug_type_extractors
             if is_type(bug, bug_map)
         ]
+
+
+class FilePaths(SingleBugFeature):
+    """Extract file paths (partial and full) from bug data."""
+
+    name = "Extract File Paths"
+
+    def __init__(self):
+        self.non_file_path_keywords = [
+            "http://",
+            "https://",
+            "www.",
+            ".com",
+            ".org",
+            ".net",
+            ".edu",
+            ".gov",
+            "@",
+        ]
```

Comment:
It'd be nice to find a more generic way to exclude these

Reply:
I'm now using `publicsuffix2` to compile all public tlds: f7e250899ee319e30dcbd70bd9942f8a63d4b59c